frontend/public/components/_monitoring.scss

```diff
@@ -44,6 +48,11 @@ $monitoring-line-height: 18px;
   padding: 0;
 }
 
+.btn--silence-remove {
```

Comment:
Also not a great name as `silence-remove` is more of an element name, not a modifier name.  I'd use just a className that describes the button itself.  I realize you're following an existing convention in this file (line 47), but it seems like a good time to clean up both instances.

Reply:
Changed to `monitoring-silence-remove-button`.

I also realized that `btn--silence-add-more` is no longer used, so removed that.